packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## v8.3.0-dev (Nov 05, 2025)
+- [Bugfix] Fix Forgot Password link not working from Account Profile password update form [#3487](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3487)
```

Comment:
nit: updated PR number in changelog
```suggestion
- [Bugfix] Fix Forgot Password link not working from Account Profile password update form [#3493](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3493)
```


Also, Should we close PR #3487 as a duplicate?